OpenTerm.xcodeproj/project.pbxproj

```diff
@@ -560,12 +564,12 @@
 				CODE_SIGN_ENTITLEMENTS = OpenTerm/OpenTerm.entitlements;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 6G5LMQ72D8;
+				DEVELOPMENT_TEAM = 2952RH3AR7;
```

Comment:
Please don't check in changes to development team or bundle identifier.

Reply:
Got it. Thanks.

Reply:
Fixed.